src/core/tests/frontend/paddle/test_models/gen_scripts/generate_not_equal.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# not_equal paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import sys
+
+
+def not_equal(name: str, x, y):
+    import paddle
+    paddle.enable_static()
+
+    node_x = paddle.static.data(name='x', shape=x.shape, dtype='float32')
+    node_y = paddle.static.data(name='y', shape=y.shape, dtype='float32')
+
+    out = paddle.not_equal(node_x, node_y)
+    out = paddle.cast(out, np.float32)
+
+    cpu = paddle.static.cpu_places(1)
+    exe = paddle.static.Executor(cpu[0])
+    # startup program will call initializer to initialize the parameters.
+    exe.run(paddle.static.default_startup_program())
+
+    outs = exe.run(
+        feed={'x': x, 'y': y},
+        fetch_list=[out])
+
+    saveModel(name, exe, feedkeys=['x', 'y'], fetchlist=[out],
+              inputs=[x, y], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def main():
+    import paddle as paddle
+    data_x = np.array([[[[-1, 0, 1]], [[2, 3, 4]]]]).astype(np.float32)
+    data_y = np.array([[[[2, 0, 3]], [[3, 1, 4]]]]).astype(np.float32)
```

Comment:
looks like this Paddle Op supports 'float32，float64，int32，int64' data type, so except 'float64' is limited by current test framework, we would better cover its supported data type as much as possible(like ['greater_than' test case](https://github.com/openvinotoolkit/openvino/blob/d22c429d0e567b80f1c5cbcde50b830fbbfac029/src/core/tests/frontend/paddle/test_models/gen_scripts/generate_greater_than.py#L43))

Reply:
updated accordingly